rosdep/base.yaml

```
@@ -6596,6 +6596,32 @@ lz4:
   rhel: [lz4-devel]
   slackware: [lz4]
   ubuntu: [liblz4-dev]
+m2r:
+  alpine:
+    pip:
+      packages: [m2r]
+  arch:
+    pip:
+      packages: [m2r]
+  debian:
+    '*': [m2r]
+    buster:
+      pip:
+        packages: [m2r]
+  fedora: [python3-m2r]
+  gentoo:
+    pip:
+      packages: [m2r]
+  nixos: [m2r]
+  opensuse: [python3-m2r]
+  osx:
+    pip:
+      packages: [m2r]
+  ubuntu:
+    '*': [m2r]
+    bionic:
+      pip:
+        packages: ['m2r<0.3']
```

```suggestion
        packages: ['m2r']
```

This form will not error during `rosdep install` but rosdep does not support detecting this as installed. If the "current" version of m2r does not work on Bionic and there is no packaged version, I think the best thing is leaving it undefined.